resources/configurationFiles/analysisRNAseq.xml

```diff
@@ -120,6 +120,8 @@
         <cvalue name='GENOME_KALLISTO_INDEX' value='${indexDirectory}/kallisto/kallisto-0.43.0_1KGRef_Gencode19_k31/kallisto-0.43.0_1KGRef_Gencode19_k31.noGenes.index' type='string'/>
         <cvalue name='ARRIBA_KNOWN_FUSIONS' value='${hg19BaseDirectory}/tools_data/arriba/known_fusions_CancerGeneCensus_gencode19_2017-01-16.tsv.gz' type='string'/>
         <cvalue name='ARRIBA_BLACKLIST' value='${hg19BaseDirectory}/tools_data/arriba/blacklist_hs37d5_gencode19_2017-01-09.tsv.gz' type='string'/>
```

Comment:
There are more recent versions of the blacklist and known fusions files that should be used with Arriba version 1.2.0. I did not notice this in the previous review, sorry. They weren't displayed in the diff ...

These are the correct versions, both of which already reside in ngs_share:

- `blacklist_hg19_hs37d5_GRCh37_2018-11-04.tsv.gz`
- `known_fusions_CancerGeneCensus_gencode19_2017-05-11.tsv.gz`

Everything else looks fine to me.

Reply:
Perfect that you noticed! I was not aware of this. I will add the new data as default. The files are, however, not part of this repo. So I will only mention them here in the XML and in the readme, such that users can install them themselves. I am very grateful for every comment, in particular if it is such a valuable one.